src/prototype-web/chart/src/chart.js

```diff
@@ -0,0 +1,51 @@
+import Chart from 'highcharts/es-modules/Core/Chart/Chart.js';
```

Comment:
Clarify on highcharts licencing please
This is not an open source software, so I'm not quite of it's usage here.